src/server/scripts/EasternKingdoms/ZulGurub/boss_thekal.cpp

```diff
@@ -34,7 +34,7 @@ enum Spells
     SPELL_MORTALCLEAVE        = 22859,
     SPELL_SILENCE             = 22666,
     SPELL_TIGER_FORM          = 24169,
-    SPELL_RESURRECT           = 24173,
+    SPELL_RESURRECTION        = 24341,
```

Comment:
I think this spell change is wrong. Was it sniffed?

Reply:
No, but the original spell doesn't actually work. If I learn it myself, it works out of combat. But if I try to use it in combat, I get a "wrong area" message and nothing happens. I suspect this is why it wasn't working for the boss before - there was no effect.

Reply:
On a side note, I'm actively working on leveling a DK on classic so I can go sniff these myself. Just not there yet.

Reply:
Then we fix the spell. Not hack it by using another spell :D

Reply:
Okay. It's just a spell effect, but I understand. I'll see if I can talk someone into getting me a sniff of the fight.